### PR TITLE
markup and styling for sidebar footer

### PR DESCRIPTION
**Pull request by Hornebom**
I think a line on top of the footer would align it more to the rest of the design

### CODE REVIEW

File: src/heads-up/components/app-sidebar.vue [hunk 1]
```diff
@@ -95,4 +105,30 @@ export default {
...
   }
+
+  .sidebar__socials {
+    margin-bottom: 20px;
+  }
+
+  .sidebar__footer {
+    margin-top: auto;
+    padding: 10px;
+    z-index: 10;
```
Comment (vbvnvmfjg) [inline src/heads-up/components/app-sidebar.vue]: It also works without `z-index`. Can you remove this?

File: src/heads-up/components/app-sidebar.vue [hunk 2]
```diff
@@ -95,4 +105,30 @@ export default {
...
+
+    color: var(--label-color);
+    text-align: center;
+    background-color: var(--toolbar-bg-color);
```
Comment (vbvnvmfjg) [inline src/heads-up/components/app-sidebar.vue]: It also works without `background-color`. Can you remove this?

File: src/heads-up/components/app-sidebar.vue [hunk 3]
```diff
@@ -95,4 +105,30 @@ export default {
     height: 1em;
     fill: currentColor;
   }
+
+  .sidebar__socials {
```
Comment (vbvnvmfjg) [inline src/heads-up/components/app-sidebar.vue]: I don't think a `margin-bottom` is necessary. You can also achieve this by setting a higher padding-top value on the `.sidebar__footer`.

File: src/heads-up/components/app-sidebar.vue [hunk 4]
```diff
@@ -95,4 +105,30 @@ export default {
...
+    margin-top: auto;
+    padding: 10px;
+    z-index: 10;
+
+    color: var(--label-color);
+    text-align: center;
+    background-color: var(--toolbar-bg-color);
+  }
+
+  .sidebar__footer-anchor {
```
Comment (vbvnvmfjg) [inline src/heads-up/components/app-sidebar.vue]: In the codebase the name `anchor` isn't used, but the name link is.
I think it's more consistent to use `.sidebar__footer-link`.

File: src/heads-up/components/app-sidebar.vue [hunk 5]
```diff
@@ -36,6 +36,12 @@
         </li>
       </ul>
     </section>
+    <footer class="sidebar__footer">
+      Made with love by 
+      <external-link class="sidebar__footer-anchor" href="https://www.voorhoede.nl" target="_blank">
```
Comment (vbvnvmfjg) [inline src/heads-up/components/app-sidebar.vue]: Can you remove `target="_blank"`? This is already used in the `<external-link>` component.

**Comment by Hornebom** [comment]
changes done

**Review by jbmoelker — APPROVED** [review]

**Pull request merged**